generators/app/index.js

```diff
@@ -434,6 +434,12 @@ module.exports = class extends BaseGenerator {
                                         this.gitInitialized = true;
                                     });
                                 }
+
+                                // gitInitialized needs to be set if already initialized
+                                // gitDir has a line break to remove (at least on windows)
```

Comment:
why not change the if above to `if (!gitDir.trim())` and do this in else, that looks cleaner IMO

Reply:
Fair enough.  I was just trying to be clear what was happening.